packages/RichTextEditing-Tests.package/RichTextEditorTestCase.class/instance/menuItemWithLabelExists..st

```diff
@@ -0,0 +1,8 @@
+utilities
+menuItemWithLabelExists: aLabel
+
+	| labels |
+	labels := OrderedCollection new.
+	self fileMenu submorphsDo: [:morph | labels add: morph contents].
+	self documentMenu submorphsDo: [:morph | labels add: morph contents].
```

Comment:
I think collect is what we want here, so something like the following. Then we might also reduce this to one return statement containing the union and the anySatisfy
```suggestion
	labels := (self fileMenu submorphs collect: [:morph | morph contents]) union:
	    (self documentMenu submorphs collect: [:morph |morph contents]).
```